localstack/testing/pytest/fixtures.py

```diff
@@ -1075,7 +1075,7 @@ def _inner():
 
             check_set = aws_client.cloudformation.describe_change_set(**kwargs)
 
-            if check_set.get("ExecutionStatus") == "ROLLBACK_COMPLETE":
+            if check_set.get("ExecutionStatus") in {"ROLLBACK_COMPLETE", "EXECUTE_FAILED"}:
```

Comment:
`ROLLBACK_COMPLETE` does not seem to be a valid execution status ever, which is why I changed this already in #8220 . Sorry for the merge conflict :)

Reply:
Haha thanks, I'll remove this change then :+1: